index.html

```diff
@@ -0,0 +1,309 @@
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>2015</li>
+              </ul>
+              <p class="section-info">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="program-btns">
+                <li class="btn-ctn">
+                  <button type="submit" class="btn-html">html</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-css">css</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-js">javascript</button>
+                </li>
+              </ul>
+              <button type="submit" class="btn4">See Project</button>
+            </li>
+            <li class="card">
+              <div class="section-img">
+                <img
+                  src="./image/images-main/snapshootportfolio4.png"
+                  alt="myproject"
+                />
+              </div>
+              <h2 class="title">multi-Post stories</h2>
+              <ul class="info">
+                <li>canopy</li>
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>back end dev</li>
+                <li>
+                  <img src="./image/images-main/Counter.png" alt="counter" />
+                </li>
+                <li>2015</li>
+              </ul>
+              <p class="section-info">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="program-btns">
+                <li class="btn-ctn">
+                  <button type="submit" class="btn-html">html</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-css">css</button>
+                </li>
+                <li class="btn-ctn">
+                  <button type="submit" class="btn btn-js">javascript</button>
+                </li>
+              </ul>
+              <button type="submit" class="btn4">See Project</button>
+            </li>
+          </ul>
+        </section>
+        <section class="about-section">
+          <div class="about-head">
+            <h2 class="about-title">
+              About<br />
+              Myself
+            </h2>
+            <p class="about-info">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="#" class="media-info">LET'S CONNECT</a>
```

Comment:
- [ ] Kindly follow the Semantic HTML concepts and the project requirements on wrapping this header in a `<hx>` tag.

Reply:
Rectify accordingly